lib/components/form.js

```diff
@@ -0,0 +1,14 @@
+import { mergeData } from "../utils";
+
+export default {
+    functional: true,
+    props: {
+        inline: {
+            type: Boolean,
+            default: false
+        }
+    },
+    render(h, { props, data, children }) {
+        return h("form", mergeData(data, { class: { "form-inline": props.inline } }), children);
+    }
+};
```

Comment:
Thi will probably conflict with the updated form.vue component in the penndig form-validation branch